pyproject.toml

```diff
@@ -172,7 +215,6 @@ include-package-data = false
 'mne_lsl.stream_viewer.control_gui.settings' = ['*.ini']
 
 [tool.setuptools.packages.find]
-exclude = ['mne_lsl*tests']
```

Comment:
Not sure why these were excluded? We include them in MNE IIRC, and excluding them makes it difficult to test the installed wheel